### ISSUE TITLE
Front.java:37-39: Document usage of Front interface in...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/takes#781]
The puzzle `373-b5720fa6` from #373 has to be resolved:

https://github.com/yegor256/takes/blob/db5df10a3e25863ca91e456e17e16a6810c357c8/src/main/java/org/takes/http/Front.java#L37-L39

The puzzle was created by Tolegen Izbassar on 26-Jan-18. 

Estimate: 15 minutes, role: IMP.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#781](https://github.com/yegor256/takes/issues/781) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Job [#781](https://github.com/yegor256/takes/issues/781) is already in scope

**Comment by 0crat** [comment]
The job [#781](https://github.com/yegor256/takes/issues/781) assigned to @t-izbassar[/z](https://www.0crat.com/u/t-izbassar), here is [why](https://www.0crat.com/footprint/C3FFK3YAY/472015194). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by t-izbassar** [comment]
@0crat waiting

**Comment by 0crat** [comment]
> @0crat waiting  ([here](https://github.com/yegor256/takes/issues/781#issuecomment-366875177))

@t-izbassar The impediment for [#781](https://github.com/yegor256/takes/issues/781) was registered successfully by @t-izbassar[/z](https://www.0crat.com/u/t-izbassar)

**Comment by 0pdd** [comment]
The puzzle `373-b5720fa6` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job, as in [§30](http://datum.zerocracy.com/pages/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by elenavolokhova** [comment]
@t-izbassar According to our [QA rules](http://datum.zerocracy.com/pages/policy.html#42)
> Some of ticket messages must mention all pull requests where the problem was actually fixed.

Please, **confirm** that you will include the link to related PR directly in next tickets.

**Comment by t-izbassar** [comment]
@elenavolokhova it's already linked by github. And commit messages that fixed an issue are indeed linked to the issue. What else is expected?

**Comment by elenavolokhova** [comment]
@t-izbassar Something like "Fixed in #803" would be great. 
Github will add links to different issues and PRs, where this issue will be just mentioned. In such cases it is hard to follow all the logic and find needed PR. It will consume some time and extra effort. So having direct mention of fixing PR is stated in QA rules. I hope it sounds reasonable)

**Comment by t-izbassar** [comment]
@elenavolokhova got it, thanks.
>Please, confirm that you will include the link to related PR directly in next tickets.

Confirmed

**Comment by elenavolokhova** [comment]
@t-izbassar Thank you!

**Comment by elenavolokhova** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
QA review completed: +15 points just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova), total is [+30](https://www.0crat.com/u/elenavolokhova)

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +30 points just awarded to @t-izbassar[/z](https://www.0crat.com/u/t-izbassar), total is [+345](https://www.0crat.com/u/t-izbassar)
